packages/happn-cluster/lib/services/proxy.js

```diff
@@ -47,12 +47,13 @@ Proxy.prototype.start = require('util').promisify(function (callback) {
     return 55000;
   };
 
-  try {
-    port = typeof this.config.port !== 'undefined' ? this.config.port : defaultPort();
-    host = dface(this.config.host);
-  } catch (err) {
-    callback(err);
+  port = typeof this.config.port !== 'undefined' ? this.config.port : defaultPort();
+  if (this.config.port === 0) {
+    port = await this.happn.services.utils.getFreePort();
```

Comment:
are we sure just using 0 for the port wont work?

Reply:
It works. Just struggled to get the port number out of proxy - issue now resolved